### PR TITLE
added jobs for automatic creation of jBPM-installers

### PR DESCRIPTION
**Pull request by mbiarnes**

### CODE REVIEW

**Comment by mbiarnes** [comment]
only makes sense with: https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/551

File: job-dsls/jobs/kie_release_jobs.groovy [hunk 1]
```diff
@@ -737,11 +737,12 @@ job("copyBinariesToFilemgmt-kieReleases-${kieVersion}") {
 
     steps {
         environmentVariables {
-            envs(MAVEN_OPTS : "${mvnOpts}", MAVEN_HOME : "\$${mvnHome}", MAVEN_REPO_LOCAL : "/home/jenkins/.m2/repository", PATH : "\$${mvnHome}/bin:\$PATH")
+            envs(MAVEN_OPTS : "${mvnOpts}", MAVEN_HOME : "\$${mvnHome}", MAVEN_REPO_LOCAL : "/home/jenkins/.m2/repository", PATH : "\$WORKSPACE/droolsjbpm-build-bootstrap/script/release:\$${mvnHome}/bin:\$PATH")
```
Comment (psiroky) [inline job-dsls/jobs/kie_release_jobs.groovy]: This looks suspicious. Why exactly is it needed?

Comment (mbiarnes) [reply job-dsls/jobs/kie_release_jobs.groovy]: A script X calls another script Y. I didn't found another way to add this PATH variable for all scripts being present. If I don't use this variable scripts X ececutes but script Y is not found. Do you know a better way?

Comment (psiroky) [reply job-dsls/jobs/kie_release_jobs.groovy]: I think we should either specify an absolute path or make sure the caller (X) can figure out precisely where the Y is located. Which script is this?

**Pull request merged**